Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -6932,14 +6932,19 @@ These ItemGrids are available in the API and can be manipulated by the plugins,
 			},
 			Variables =
 			{
+				m_BookContent =
+				{
+					Type = "{{cBookContent|cBookContent}}",
```

Comment:
This `Type = "cBookContent"` doesn't work for member variables. have to use `{{cBookContent}}`. Will look into APIDump what there is wrong.